registry.yaml

```diff
@@ -1,358 +1,134 @@
   description: Load-test SQL Servers
   tier: official
   imports:
     - k6/x/sql
-  constraints: ">=1.0.0"
-- module: github.com/grafana/xk6-sql-driver-azuresql
-  description: xk6-sql driver extension for Microsoft Azure SQL database support
-  imports:
-    - k6/x/sql/driver/azuresql
-  constraints: ">=0.1.0"
-- module: github.com/grafana/xk6-sql-driver-clickhouse
-  description: xk6-sql driver extension for ClickHouse database support
-  imports:
-    - k6/x/sql/driver/clickhouse
-  constraints: ">=0.1.0"
+  versions:
+    - "1.0.0"
+    - "1.0.1"
+    - "1.0.2"
+    - "1.0.3"
+    - "1.0.4"
+    - "1.0.5"
+
 - module: github.com/grafana/xk6-sql-driver-mysql
   description: xk6-sql driver extension for MySQL database support
   tier: official
   imports:
     - k6/x/sql/driver/mysql
-  constraints: ">=0.1.0"
+  versions:
+    - "0.1.0"
+    - "0.2.0"
+    - "0.2.1"
+
 - module: github.com/grafana/xk6-sql-driver-postgres
   description: xk6-sql driver extension for Postgres database support
   tier: official
   imports:
     - k6/x/sql/driver/postgres
-  constraints: ">=0.1.0"
-- module: github.com/grafana/xk6-sql-driver-sqlite3
-  description: xk6-sql driver extension for SQLite v3 database support
-  tier: official
-  cgo: true
-  imports:
-    - k6/x/sql/driver/sqlite3
-  constraints: ">=0.1.0"
-- module: github.com/grafana/xk6-sql-driver-sqlserver
-  description: xk6-sql driver extension for Microsoft SQL Server database support
-  imports:
-    - k6/x/sql/driver/sqlserver
-  constraints: ">=0.1.0"
-- module: github.com/grafana/xk6-sql-driver-ramsql
-  description: xk6-sql driver extension for RamSQL database support
-  imports:
-    - k6/x/sql/driver/ramsql
-  constraints: ">=0.1.0"
+  versions:
+    - "0.1.0"
+    - "0.1.1"
+
 - module: github.com/grafana/xk6-ssh
   description: Use SSH connections in your tests
   tier: official
   imports:
     - k6/x/ssh
-  constraints: ">=0.1.0"
-- module: github.com/kelseyaubrecht/xk6-webtransport
-  description: Add support for webtransport protocol
-  imports:
-    - k6/x/webtransport
-- module: github.com/kubeshop/xk6-tracetest
-  description: Support for Tracetest test execution and tracing client
-  outputs:
-    - tracetest
-  constraints: ">=0.1.10"
-- module: github.com/leonyork/xk6-output-timestream
-  description: Export results to AWS Timestream
-  outputs:
-    - timestream
-  constraints: ">=0.9.80"
-- module: github.com/maksimall89/xk6-telegram
-  description: Interact with Telegram Bots
-  imports:
-    - k6/x/telegram
-- module: github.com/martymarron/xk6-output-prometheus-pushgateway
-  description: Export results to Prometheus pushgateway
-  imports:
-    - k6/x/output-prometheus-pushgateway
-  outputs:
-    - output-prometheus-pushgateway
+  versions:
+    - "0.1.0"
+    - "0.1.1"
+    - "0.1.2"
+    - "0.1.3"
+
+# Community extensions
+
 - module: github.com/mostafa/xk6-kafka
   description: Load test Apache Kafka. Includes support for Avro messages.
+  tier: community
```

Comment:
@pablochacin I noticed these were all missing `tier: community` but not sure if this addition is correct

Reply:
It's the dafault, but adding won't hurt